truffle-config.js

```diff
@@ -1,3 +1,5 @@
+require('babel-register')
```

Comment:
what is this doing?? :)

Reply:
so we can use async/await in tests .. we had it in the starter project but this config was copied over from zeppelin